### DISCUSSION TITLE
Darkmode

### DISCUSSION THREAD

**Discussion by PatrickLaabs** [maragudk/gomponents#138] [Q&A] [answered]
Hey everyone,
I am just getting started with gomponents, and so far I am loving it :) 

I am just wondering, how can I either force to always have my website being shown in the "darkmode", or
how I might be able to switch images, depending of the current mode I have.

Like:
If I am in "Light-Mode" use Image/Logo XX, and if in "Dark-Mode" use Image/Logo YY.

Has anyone came across a solution to this?

Best
Patrick

**Comment by markuswustenberg** [comment, marked as answer]
Hey @PatrickLaabs, I'm happy to hear you like the library! 😊

This isn't necessarily particular to this library, but something you can do in CSS directly. I often use TailwindCSS for this, which has built-in support for dark mode: https://tailwindcss.com/docs/dark-mode

With just gomponents, you would need some kind of state server-side and render different components based on that state.

Hope it helps!

**Reply by PatrickLaabs** [reply]
Hey and thanks for the reply.
Alrighty, I will take a closer look at the way Tailwind handles this.
I guess, it is something obvious, that I might not see yet. 

By the way - but maybe a little off-topic:
Are there any informations regarding the GDPR stuff we have to deal with in Europe? :)

**Reply by markuswustenberg** [reply]
GDPR related to what?

**Reply by PatrickLaabs** [reply]
Cookies, Tracking, etc. But, as far I can see, the website is not tracking anything, which might make it gdpr compliant per default?

Oh, and btw:
The more I play around with your library, the more fascinated I am. I almost got the hang of it, and it becomes almost natural to build the site with gomponents.. thats pretty awesome.

**Reply by markuswustenberg** [reply]
@PatrickLaabs I'm a bit unsure which website you mean? 😅

I'm happy you like it! I'm pretty happy with it as well. And it becomes more valuable over time when you build your component collection.
